src/Explorer/Explorer.ts

```diff
@@ -1953,6 +1956,16 @@ export default class Explorer {
     return Q();
   }
 
+  public setFeatureFlagsFromFlights(flights: readonly string[]): void {
+    if (!flights) {
+      return;
+    }
+
+    if (flights.filter((flightName: string) => flightName === Constants.Flights.settingsV2).length === 1) {
+      this.isSettingsV2Enabled(true);

```

Comment:
>isSettingsV2Enabled [](start = 11, length = 19)

as discussed yesterday, do you want to turn it on for MPAC - just for this feature so we can find issues faster?